sensor_msgs/PointCloud2 point_cloud2
---
view_point_extractor/ViewPoint[] view_point_vec
